src/main/java/movieMaker/MovieWriter.java

```diff
@@ -220,8 +218,8 @@ public List getFrameNames()
   public boolean writeAVI()
   {
     JpegImagesToMovie imageToMovie = new JpegImagesToMovie();
-    List frameNames = getFrameNames();
-    Picture p = new Picture((String) frameNames.get(0));
+    List<String> frameNames = getFrameNames();
```

Comment:
what happens if frameNames is empty?

Reply:
Nothing good.
Not sure how a user could cause it to happen easily, but I added a check.

Reply:
looks like you only got one of the two...

Reply:
What!? Two places!?
You want me to look at all thirty changed lines and figure that out?

(Oops. Thanks for catching that.)

Fixed.